### ISSUE TITLE
[Bug] Using VisualStateManager to change ImageButton source when pressed

### ISSUE THREAD

**Issue by Informatheus** [xamarin/Xamarin.Forms#9428]
### Description
Using VisualStateManager to switch the source of an ImageButton during the pressed state.

### Steps to Reproduce
1. Use 2 equal images to become sources of the ImageButton.
2. Use a VisualStateManager to set one image while Normal, and another while Pressed.

### Expected Behavior
ImageButton changes its source image while pressed and returns to first image when released.

### Actual Behavior
- The result on Android is a kind of bug where after pressing the image it becomes much smaller than the button, in the upper left corner of ImageButton and does not return to normal.
- On Windows, the first image only comes back when I click outside the ImageButton.
- I have not how to test on iOS for now.

### Basic Information
- Version with issue: I have installed VS and Xamarin last week.
- IDE: Visual Studio
- Platform Target Frameworks: 
- Android: API 28
- Windows 10
- Nuget Packages: only Github for VS.
- Affected Devices: Genymotion and a real Redmi Note 7

### Code

```
<ImageButton HeightRequest="200" WidthRequest="200"
                         VerticalOptions="CenterAndExpand" 
                         HorizontalOptions="CenterAndExpand"
                         BackgroundColor="Transparent">

                <VisualStateManager.VisualStateGroups>
                    <VisualStateGroup x:Name="button">
                        <VisualState x:Name="Normal">
                            <VisualState.Setters>
                                <Setter Property="Source" Value="button.png" />
                            </VisualState.Setters>
                        </VisualState>

                        <VisualState x:Name="Pressed">
                            <VisualState.Setters>
                                <Setter Property="Source" Value="button2.png" />
                            </VisualState.Setters>
                        </VisualState>

                    </VisualStateGroup>
                </VisualStateManager.VisualStateGroups>
</ImageButton>
```
### Screenshots
Before click:
![image](https://user-images.githubusercontent.com/3857461/73799819-ca6c0f00-4795-11ea-899c-fcfb35cb42cd.png)

During click:
![image](https://user-images.githubusercontent.com/3857461/73799880-f1c2dc00-4795-11ea-8ae9-97528eb13ca2.png)

After click:
![image](https://user-images.githubusercontent.com/3857461/73799898-fdae9e00-4795-11ea-9146-6995e5a5b815.png)

**Comment by PureWeen** [comment]
@Informatheus 
https://github.com/xamarin/Xamarin.Forms/pull/9661

For UWP what's happening there is that the state is transitioning to "Focused" and then when you click off it's unfocused. Because focus doesn't have anything defined it just stays in the "Normal" state

If you add a focused state
```XAML
  <VisualStateManager.VisualStateGroups>
                    <VisualStateGroup x:Name="button">
                        <VisualState x:Name="Normal">
                            <VisualState.Setters>
                                <Setter Property="Source" Value="coffee.png" />
                            </VisualState.Setters>
                        </VisualState>
                        <VisualState x:Name="Focused">
                            <VisualState.Setters>
                                <Setter Property="Source" Value="coffee.png" />
                            </VisualState.Setters>
                        </VisualState>
                        <VisualState x:Name="Pressed">
                            <VisualState.Setters>
                                <Setter Property="Source" Value="calculator.png" />
                            </VisualState.Setters>
                        </VisualState>

                    </VisualStateGroup>
                </VisualStateManager.VisualStateGroups>
```

Then it works on UWP

**Comment by samhouts** [comment]
closed by #9661
